app/javascript/app/components/countries-menu/countries-menu-styles.scss

```diff
@@ -0,0 +1,43 @@
+@import '~styles/layout.scss';
+
+.fixed {
+  position: fixed;
```

Comment:
You cant used fixed for something like this. Try scrolling down the page to see the effect. It needs to be absolute, or cleverly made relative to the top bar.